scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -4,19 +4,25 @@
 
 public interface ScalafixEvaluation {
     /**
-     * @return boolean true if scalafix has successfully been evaluated on all files configured and no ScalafixError has been found
+     * @return boolean true if scalafix has successfully been evaluated on all files configured with no error.
      */
     boolean isSuccessful();
 
     ScalafixError[] getErrors();
 
+    /**
+     *
+     * @return a more detailed error message that can be
```

Comment:
more than what?

Reply:
the error , wchich is only the type of the Enum